wiki/en/en-Client-Troubleshooting.md

```diff
@@ -12,6 +12,18 @@ Start with the simple stuff: make sure your instrument/microphone and headphones
 
 **Windows users (ASIO4ALL)**: If you’re using the ASIO4ALL driver have a look at the [ASIO4LL setup section](Installation-for-Windows#setting-up-asio4all)
 
+### Getting a loud squeal if you connect to a server
+
+You're creating feedback since your microphone is recording the signal of your speaker. Disconnect from the server and select the correct input and output devices. See the installation page for your OS to get more information on how to do that.
+
+### Not seing the headphone or microphone you just connected to your PC?
+
+Jamulus currently doesn't show devices which were plugged in after the sound system was loaded. Just restart Jamulus with your device plugged in.
```

Comment:
I'd swap these two sentences around, and consider not having the second. Non-engineers don't care why something is happening in this case, only how to fix it :-)

Reply:
Done.